test/e2e/framework/pv_util.go

```diff
@@ -575,17 +590,32 @@ func makePvcKey(ns, name string) types.NamespacedName {
 //   is added later in CreatePVCPV.
 func MakePersistentVolume(pvConfig PersistentVolumeConfig) *v1.PersistentVolume {
 	var claimRef *v1.ObjectReference
```

Comment:
What do you think about moving this declaration down to line 609? (as low as it can go and still compile)

Reply:
there are other functions higher up in this file that use PVConfig